va/http.go

```diff
@@ -635,6 +640,15 @@ func (va *ValidationAuthorityImpl) processHTTPValidation(
 		return nil, records, berrors.UnauthorizedError("Invalid response from %s [%s]: %q",
 			records[len(records)-1].URL, records[len(records)-1].AddressUsed, replaceInvalidUTF8(body))
 	}
+
+	if httpResponse.TLS != nil && httpResponse.TLS.Version < tls.VersionTLS12 {
```

Comment:
I think this needs to be captured before the response body is closed on line `629`. The docs for the `Response` object state that the pointer for this field is shared between responses.